lib/rack/default_content_type.rb

```diff
@@ -60,8 +60,9 @@ def call(env)
 
     # :reek:UtilityFunction
     # :reek:NilCheck
+    # :reek:DuplicateMethodCall
     def content_type_missing?(env)
-      env['CONTENT_TYPE']&.empty?
+      env['CONTENT_TYPE'].nil? || env['CONTENT_TYPE'].empty?
```

Comment:
I believe you could just make this `env['CONTENT_TYPE'].blank?` or you could use `!env['CONTENT_TYPE'].present?`. `nil` should respond to both of those without a `NoMethodError`.

Reply:
@jonahx has commented on that before, that we shouldn't use the Rails helpers in Rack middleware. Otherwise `blank?` would be the correct solution.

Reply:
These reek UtilityFunction callouts look valid to me.  

There are two options, I'll start with the one I'd recommend.

1.  Remove the methods and inline their logic in `call` -- they are both one-liners anyway.  Assign the result of what the calculate to a local var corresponding to the current method names. 

2. Create an object which is initialized with the `env`, and then the references to `@env` will be references to an instance var. 

This is a good equivalence to keep in mind generally.  And generally, the decision between 1. and 2. comes down to... "If complex enough (more than a few methods) go with 2.  Otherwise 2. is overkill so go with 1."